### PR TITLE
port the feedback script to Go [#127883845] [#127967963]

### PR DESCRIPTION
**Pull request by kdomanski**

### CODE REVIEW

**Comment by colszowka** [comment]
Very nice, but please add tests :heart:

**Comment by colszowka** [comment]
Regarding the canFail-Flag: I'd argue all of them should be allowed to fail and the other data should remain collected

File: go/platform-feedback/main.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+	getCommandsOutput(false, dir, "zpool-events", "zpool", "events")
+	getCommandsOutput(false, dir, "zfs-list", "zfs", "list")
+	getCommandsOutput(false, dir, "zfs-get-all", "zfs", "get", "all")
+
+	archiveName, err := tarTheData(dir)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	os.RemoveAll(dir)
```
Comment (colszowka) [inline go/platform-feedback/main.go]: Maybe move it up and do a `defer` to ensure it actually happens like in the other cases?

**Comment by colszowka** [comment]
Let's please also add `/data/collectd` into this

**Comment by kdomanski** [comment]
@colszowka There is a separate ticket for `collectd` I'm working on right now. Re-read the code and please try to figure out what the `canFail` flag actually does.

**Comment by colszowka** [comment]
Where the f did our comments from yesterday go!?

**Pull request merged**